### PR TITLE
[WIP] Upgrade GitLab to 12.1.0

### PR DESCRIPTION
**Pull request by solidnerd**

### CODE REVIEW

**Comment by TimeBye** [comment]
sameersbn/docker-gitlab 12.1.0 still support mysql?

https://gitlab.com/gitlab-org/gitlab-ce/blob/master/doc/install/installation.md#6-database

**Comment by drujd** [comment]
Yep, I think you can remove all the MySQL packages. Maybe add a warning to migrate to postgresql before upgrading?

**Comment by drubin** [comment]
All references/examples for mysql should be removed too.

**Comment by solidnerd** [comment]
If aynone wants to do it before next week feel free. I will have time for
that next week.

I could bei targeted in this pr.

David Rubin <notifications@github.com> schrieb am Mo., 29. Juli 2019, 21:16:

**Comment by paolomainardi** [comment]
Thanks @solidnerd, so guys it's our turn, can we help @solidnerd on that ?

**Comment by Ilya-Kuchaev** [comment]
Any chance to see it merged?

**Comment by Ilya-Kuchaev** [comment]
Ping @solidnerd @sameersbn

**Comment by solidnerd** [comment]
Hey,
I think I'll be finished by end of week. I did the following steps:
1. Update base image to bionic
2. Remove MySQL Support

I have currently a problem with pax and ruby2.6. I think it's needs also to be builded from source instead of a package to get this solved.

**Comment by solidnerd** [comment]
The current progress is here: https://github.com/solidnerd/docker-gitlab/tree/prepare-remove-mysql

**Comment by solidnerd** [comment]
LGTM !

**Comment by lanmingle** [comment]
very good

File: assets/build/install.sh [hunk 1]
```diff
@@ -153,15 +153,15 @@ if [[ -d ${GEM_CACHE_DIR} ]]; then
   chown -R ${GITLAB_USER}: ${GITLAB_INSTALL_DIR}/vendor/cache
 fi
 
-exec_as_git bundle install -j"$(nproc)" --deployment --without development test aws
+exec_as_git bundle install -j"$(nproc)" --deployment --without development test mysql aws
```
Comment (splattael) [inline assets/build/install.sh]: @solidnerd I was wondering why we need to install the `mysql` gem :thinking:

**Pull request merged**